.github/auto-assign_config.yml

```diff
@@ -0,0 +1,10 @@
+addReviewers: true
+
+addAssignees: author
```

Comment:
```suggestion
```
It seems redundant to me to add the author as an assignee to their own PR

Reply:
This seems logical to me since it is he who is at the origin of the PR.

Reply:
Ok 👍